server/service/src/invoice/inbound_shipment/add_from_master_list.rs

```diff
@@ -98,7 +98,8 @@ fn generate(
         .query_by_filter(
             MasterListLineFilter::new()
                 .master_list_id(EqualFilter::equal_to(&input.master_list_id))
-                .item_id(EqualFilter::not_equal_all(item_ids_in_invoice)),
+                .item_id(EqualFilter::not_equal_all(item_ids_in_invoice))
+                .item_type(ItemRowType::Stock.equal_to()),
```

Comment:
We have to decide if KDD should be in master list and other discussions about how service charge should be handled in om since it's different in OG, that's why I said quick fix XD

Reply:
Yes thats my point, then its easy to remove.

Reply:
Think it's fine not to remove even if we remove service from master list? Since it's searching for item that are stock...